lua/hfcc/completion.lua

```diff
@@ -125,6 +125,7 @@ function M.complete()
     api.nvim_win_set_cursor(0, { row_offset, col_offset })
 
     M.suggestion = nil
+    vim.defer_fn(M.schedule, 10)
```

Comment:
noob question: could you clarify why is this needed?

Reply:
To continue generating on <accept> of suggestion.
If to execute it right away it doesn't work, because of the way functions are executed.
It defers the execution after certain amount of time(could be wrong regarding time tho)